### PR TITLE
Added Delete tank function and Delete test. 

### PR DESCRIPTION
**Pull request by Suocesky**
Added a new function and button to remove the selected tank as well as a test to verify it works. Some tests were also combined into one.

### CODE REVIEW

File: lib/components/app_drawer.dart [hunk 1]
```diff
@@ -41,6 +41,17 @@ class AppDrawer extends StatelessWidget {
               child: const Icon(Icons.add),
             ),
           ),
+          Align (
+            alignment: Alignment.topRight,
+            child: FloatingActionButton(
+              onPressed: () {
+                appData.removeTank(appData.currentTank);
+                appData.currentTank = Tank("", "");
```
Comment (jgfoster) [inline lib/components/app_drawer.dart:49]: How many places in the code do we have `Tank("", "")`? How would we remove that duplicate code? (This should probably be a separate PR, but is an example of repetition that could be simplified.)

**Review by jgfoster — CHANGES_REQUESTED** [review]
spacing issue in lib/model/tc_interface.dart.

File: lib/model/tc_interface.dart [hunk 2]
```diff
@@ -31,10 +31,10 @@ class TcMockInterface extends TcInterface {
 
 class TcRealInterface extends TcInterface {
   Future<String> get(var ip, var path) async {
```
[the remark this answers is not in the document]
Comment (Suocesky) [reply lib/model/tc_interface.dart]: The extra indentation has been removed.

File: lib/components/app_drawer.dart (hunk 1 above)
Comment (Suocesky) [reply lib/components/app_drawer.dart:49]: From what I can find, it occurs 6 times in the app and once in testing. I should be able to write a short function to set it to a default empty Tank.

**Review by jgfoster — APPROVED** [review]

**Review by prestoncarman — APPROVED** [review]
